tree/ntuple/v7/test/ntuple_compression.cxx

```diff
@@ -0,0 +1,72 @@
+#include <ROOT/RNTuple.hxx>
```

Comment:
I think a number of includes are not necessary.  Please trim down to the minimum includes list.